path.go

```diff
@@ -71,6 +72,18 @@ func getFinalRecord(zone string, from int) (record, error) {
 		return rec, fmt.Errorf("could not parse record: %s", err)
 	}
 
+	if rec.Type == "path" {
+		url, err := url.Parse(rec.To)
+		if err != nil {
+			return rec, fmt.Errorf("could not parse the url: %s", err)
+		}
+		zone, from, err := zoneFromPath(url.Host, url.Path, rec)
+		if err != nil {
+			return rec, fmt.Errorf("could not generate zone URI from path: %s", err)
+		}
+		rec, err = getFinalRecord(zone, from)
```

Comment:
I think it's because we don't return the error here on line 84. the fallback code only gets triggered when an error has been returned. am I right?